riscv/src/compiler.rs

```diff
@@ -719,20 +745,20 @@ fn preamble() -> String {
         { REM_b4 } in { bytes };
     }
 
-    // implements Z = Y / X, stores remainder in `remainder`.
+    // implements Z = Y / X, stores remainder in `div_tmp`.
     instr divu Y, X -> Z {
         // Y is the known dividend
         // X is the known divisor
         // Z is the unknown quotient
         // main division algorithm;
         // if X is zero, remainder is set to dividend, as per RISC-V specification:
-        X * Z + remainder = Y,
+        X * Z + div_tmp = Y,
```

Comment:
why div_tmp?

Reply:
In remu, it is the quotient, in divu, it is the remainder.